### DISCUSSION TITLE
Markdown table horizontal scrollbar

### DISCUSSION THREAD

**Discussion by hallerx** [requarks/wiki#3472] [Help / Questions]
**Question**
I have created a markdown table in WikiJS, which is... quite large. 
Unfortunately, I am only able to add CSS code to the editor for horizontal scrolling, but when saving and viewing 'live' page, I don't see horizontal scrollbar for the table, therefore, unable to horizontally scroll to fully view it.

Is there any way to implement horizontal scroll bar for markdown table? Maybe override CSS somehow to include in all pages horizontal scroll bar by default for whole page?

**Host Info (please complete the following information):**
OS: Ubuntu 18.04 VMware hosted Guest OS
Wiki.js version: 2.5.159
Database engine: postgresql 10.14

**Comment by Igorgro** [comment]
Duplicate of https://github.com/Requarks/wiki/issues/1314
